docs/cli/price-exporter.md

```diff
@@ -0,0 +1,103 @@
+  - `exportFormat` (String): Export format ['csv', 'json'] (Optional. Default: 'json')
+  - `predicate` (String): Query string specifying (where) predicate. More info on predicates [here](http://dev.commercetools.com/http-api.html#predicates) (Optional)
+  - `staged` (Boolean): Specify if prices should be fetched from all products (true) or only published products (false) (Optional. Default: false)
+  - `csvHeaders` (Array<String>): Array containing headers for the returned CSV price data. If omitted, all headers will be turned. (Optional)
+- An optional logger object having four functions (`info`, `warn`, `error` and `verbose`)
+
+## Usage
+`npm install @commercetools/price-exporter --global`
+
+### CLI
+```
+Usage: price-exporter [options]
+Export prices from the commercetools platform.
+
+Options:
+  --help, -h          Show help text.                                  [boolean]
+  --version           Show version number                              [boolean]
+  --input, -i         Path to CSV template.
+  --output, -o        Path to output file.                   [default: "stdout"]
+  --apiUrl            The host URL of the HTTP API service.
+                                              [default: "https://api.sphere.io"]
+  --authUrl           The host URL of the OAuth API service.
+                                             [default: "https://auth.sphere.io"]
+  --projectKey, -p    API project key.                                [required]
+  --accessToken       CTP client access token.
+  --delimiter, -d     Used CSV delimiter for template and output. [default: ","]
+  --where, -w         Where predicate for products from which to fetch prices.
+  --exportFormat, -f  Format for export.
+                                      [choices: "csv", "json"] [default: "json"]
+  --staged, -s        Specify if prices should be from all or published
+                      products.                                        [boolean]
+  --logLevel          Logging level: error, warn, info or verbose.
+                                                               [default: "info"]
+  --logFile           Path to file where to save logs.
+                                                 [default: "price-exporter.log"]
+```
+
+#### Info on flags
+- The `--input` flag specifies the path to an optional CSV template file.
+  - Only the first line is read and subsequent lines (if present) will be ignored
+  - The delimiter must match the delimiter passed in by `--delimiter` (or the default delimiter)
+- The `--output` flag specifies where to output/save the exported prices.
+  - If the file specified already exists, it will be overwritten.
+  - The default location for status report logging is the standard output.
+  - If no output path is specified, the exported prices will be logged to the standard output.
+- The `--delimiter` flag specifies the delimiter used in the output file if CSV. Defaults to `','` if omitted.
```

Comment:
I think this delimiter works both for output file and input file. We should specify that.